test/Makefile.am

```diff
@@ -29,7 +29,7 @@ t8code_test_programs = \
     test/t8_test_netcdf_linkage \
     test/t8_test_vtk_linkage
 
-test_t8_cmesh_t8_test_hypercube_SOURCES = test/t8_cmesh/t8_test_hypercube.c
```

Comment:
You have removed the file from the Makefile, but can you delete it from the repository, too?